resources/extra-large-conference/README.md

```diff
@@ -55,13 +55,21 @@ Setting up configuration for the main prosody is a manual process:
 -- targets must be IPs, not hostnames
 s2s_connect_overrides = {
     ["conference.v1.meet.jitsi"] = { "127.0.0.1", 52691 };
+    ["v1.meet.jitsi"] = { "127.0.0.1", 52691 }; -- neded for v1.meet.jitsi->visitors.jitmeet.example.com
```

Comment:
typo in "neded", also I don't see `jitmeet.example.com` referenced anywhere else, maybe just change to `visitors.example.com`?